### DISCUSSION TITLE
Great Stuff!

### DISCUSSION THREAD

**Discussion by mikecann** [ticket-bridge/hyper-durable#22] [General]
I just want to say this is really good stuff! 

I was asking about something like this on Discord and Cloudflare Matt linked to it: https://discord.com/channels/595317990191398933/773219443911819284/988578837422034944

[My attempt](https://github.com/mikecann/clouds-and-edges/blob/main/packages/workers-es/src/durableObjects/createDurableObjectRPCProxy.ts) at this a while back was very basic so im really glad to see that someone has done a MUCH better job at it!

Very well done and keep up the awesome work :)

**Comment by travismfrank** [comment]
Hey @mikecann!  Thanks for your kind words.  Cool to see you're also working in the blockchain space- at [Ticket Bridge](https://ticketbridge.io) I'm working on making unscalpable tickets (achieved by minting the tickets as NFTs).  Have you had much success with DOs at Gangbusters?

**Reply by mikecann** [reply]
Oh nice! Thats a great use of the Blockchain :) 

Currently my DO experiments have been contained to personal projects (https://mikecann.co.uk/posts/serverless-databaseless-event-sourcing). 

We have a lot of legacy code which I am trying to be pragmatic about despite how temping a rewrite is ;)
